src/analysis/utils/runtime-utils.ts

```diff
@@ -74,7 +74,9 @@ export type NormalizedComputeRegion = NominalType<string, 'ComputeRegion'>;
 // TODO: test when zone and region have types
 export const getNormalizedComputeRegion = (config: GoogleRuntimeConfig | AzureConfig): NormalizedComputeRegion => {
   if (isGceConfig(config) || isGceWithPdConfig(config)) {
-    return getRegionFromZone(config.zone).toUpperCase() as NormalizedComputeRegion;
+    if ('zone' in config) {
```

Comment:
Why do we need this extra check? Would we get GCE runtimes without a zone in its config?

Reply:
We don't need it, I had added it while I was debugging.